Makefile

```diff
@@ -54,9 +54,12 @@ precommit-install:
 precommit:
 	pre-commit run --all
 
+
+
+
```

Comment:
Whitespace change? Or was formatting off before?

Reply:
ah my b this was the test :)